.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+$finder = PhpCsFixer\Finder::create()
+  ->in(__DIR__.'/{src,tests}')
+;
+
+$header =<<<'HEADER'
+This file is part of itk-dev/serviceplatformen.
+(c) 2020 ITK Development
+This source file is subject to the MIT license.
+HEADER;
```

Comment:
We should not set the header in all files – and not to something containing “serviceplatformen”. The header does not really add any value. At least we should write `(c) 2020– ITK Development`.

Reply:
Removed